src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -1013,3 +1009,20 @@ EnsureReplicationSettings(Oid relationId)
 								"factor\" to one%s.", extraHint)));
 	}
 }
+
+
+/*
+ * DisplayReplicationModelOverwriteNotice shows a notice message to tell that we use
+ * statement based replication, even if the citus.replication_factor setting
+ * is set to streaming.
+ */
+static void
+DisplayReplicationModelOverwriteNotice(void)
+{
+	if (ReplicationModel != REPLICATION_MODEL_COORDINATOR)
+	{
+		ereport(NOTICE, (errmsg("using statement based replication"),
```

Comment:
Not sure if this notice takes away confusion or adds to it, since even advanced users likely won't be familiar with our notion of replication model and the differences are very subtle.

I'm not sure if we have anything useful to say here, but we can at least elaborate a bit. "master_create_distributed_table is deprecated and does not support the current replication_model setting"